packages/sanity/src/core/form/studio/FormBuilderInputErrorBoundary.tsx

```diff
@@ -22,14 +23,17 @@ export function FormBuilderInputErrorBoundary(
   props: FormBuilderInputErrorBoundaryProps,
 ): JSX.Element {
   const {children} = props
-  const [{error}, setError] = useState<{error: unknown}>({error: null})
-  const handleRetry = useCallback(() => setError({error: null}), [])
+  const [{error, info}, setError] = useState<{error: unknown; info: React.ErrorInfo}>({
```

Comment:
stylistic comment: since you made info optional in the ErrorCard, would it be better for `info` here to be optional as well?

Reply:
The typings from `ErrorCard` in `@sanity/ui` claims that it's always provided, so I follow that assumption here to ensure default values and reset values fulfill that contract.

It's optional in the component as a defensive measure, to ensure that should the component in `@sanity/ui` somehow return `null` or `undefined` then the ErrorCard makes sure to account for it. I don't think it's likely to happen, but given that this component is supposed to handle unhandled runtime exceptions it felt reasonable to follow Murphy's Law ☺️